src/components/frame/Extensions/layouts/hooks.tsx

```diff
@@ -357,48 +357,52 @@ export const useGetTxButtons = () => {
   const { groupState } = useUserTransactionContext();
   const isMobile = useMobile();
 
-  const txButtons = [
-    groupState === TransactionGroupStates.SomePending && (
-      <TxButton
-        text={isMobile ? undefined : { id: 'button.pending' }}
-        className={clsx({
-          '!min-w-0': isMobile,
-        })}
-        icon={
-          <span
-            className={clsx('flex shrink-0', {
-              'ml-1.5': !isMobile,
-            })}
-          >
-            <Icon
-              name="spinner-gap"
-              className="animate-spin"
-              appearance={{ size: 'tiny' }}
-            />
-          </span>
-        }
-        data-openhubifclicked // see UserReputation for usage
-      />
-    ),
-    groupState === TransactionGroupStates.AllCompleted && (
-      <TxButton
-        text={isMobile ? undefined : { id: 'button.completed' }}
-        className={clsx({
-          '!min-w-0': isMobile,
-        })}
-        icon={
-          <span
-            className={clsx('flex shrink-0', {
-              'ml-1.5': !isMobile,
-            })}
-          >
-            <Icon name="white-tick" appearance={{ size: 'tiny' }} />
-          </span>
-        }
-        data-openhubifclicked // see UserReputation for usage
-      />
-    ),
-  ];
+  const txButtons = (
+    <>
+      {groupState === TransactionGroupStates.SomePending && (
+        <TxButton
+          key="1"
```

Comment:
do we need keys here since it's no rendered in an array?

Reply:
@bassgeta yes, otherwise in browser console warning appears with missing key, this component is used in several places and in one of theme them it's rendered as an array.